homeassistant/components/notify/__init__.py

```diff
@@ -66,7 +66,12 @@ async def async_reload(hass, integration_name):
     ):
         return
 
-    data = hass.data[NOTIFY_SERVICES][integration_name]
+    for data in hass.data[NOTIFY_SERVICES][integration_name]:
+        await _async_setup_notify_services(hass, data)
```

Comment:
Can this be done concurrently with `asyncio.gather`?

Reply:
Seems to work just fine

https://github.com/home-assistant/core/pull/39384